### PR TITLE
Add FrontEnd Masters Guides

### PR DESCRIPTION
**Pull request by davorpa**
## What does this PR do?
Add resource(s)

### Why is this valuable (or not)?

Ensure adds resources found at https://frontendmasters.com/guides/

### How do we know it's really free?

CC-BY-3

### For book lists, is it a book? For course lists, is it a course? etc.

Books/Guides with exercises to track front-end web development discipline using HTML5, CSS, Javascript...

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md)
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Review by LuigiImVector — APPROVED** [review]
LGTM

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -531,6 +531,10 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Encyclopedia of Human Computer Interaction 2nd Edition](https://www.interaction-design.org/literature/book/the-encyclopedia-of-human-computer-interaction-2nd-ed)
 * [Essential Image Optimization](https://images.guide) - Addy Osmani
 * [Foundations of Programming](https://openmymind.net/FoundationsOfProgramming.pdf) - Karl Seguin (PDF)
+* [Front-End Developer Handbook 2016](https://frontendmasters.com/guides/front-end-handbook/2016/) - Cody Lindley, Frontend Masters (HTML)
+* [Front-End Developer Handbook 2017](https://frontendmasters.com/guides/front-end-handbook/2017/) - Cody Lindley, Frontend Masters (HTML)
+* [Front-End Developer Handbook 2018](https://frontendmasters.com/guides/front-end-handbook/2018/) - Cody Lindley, Frontend Masters (HTML)
+* [Front-End Developer Handbook 2019](https://frontendmasters.com/guides/front-end-handbook/2019/) - Cody Lindley, Frontend Masters (HTML, [PDF](https://github.com/FrontendMasters/front-end-handbook-2019/raw/HEAD/exports/Front-end%20Developer%20Handbook%202019.pdf), [EPUB](https://github.com/FrontendMasters/front-end-handbook-2019/raw/HEAD/exports/Front-End%20Developer%20Handbook%202019.epub))
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:537]: Only one link is needed for this book. Please remove the format specific links.

**Review by eshellman — CHANGES_REQUESTED** [review]

File: books/free-programming-books-subjects.md (hunk 1 above)
Comment (davorpa) [reply books/free-programming-books-subjects.md]: Resolved by commit 0f9aab2d717e35fee0cd0774e51e38f36f2b4eae

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -1320,7 +1320,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Learning React.js: Getting Started and Concepts](https://scotch.io/tutorials/learning-react-getting-started-and-concepts)
 * [React-Bits](https://github.com/vasanthk/react-bits)
 * [React Book, your beginner guide to React](https://github.com/softchris/react-book/) - Chris Noring
-* [React Enlightenment](https://www.reactenlightenment.com)
+* [React Enlightenment](https://www.reactenlightenment.com) - Cody Lindley, Frontend Masters (HTML)
```
[the remark this answers is not in the document]
Comment (davorpa) [reply books/free-programming-books-langs.md]: Resolved by commit: 36218cf403bbfe568a5445519af2031ca91795e1

**Comment by eshellman** [comment]
thanks!

**Pull request merged**